### ISSUE TITLE
How do i make a silent start for sndcpy?, (Not showing the console)

### ISSUE THREAD

**Issue by YacuGitHub** [Genymobile/scrcpy#4016]
I know this is the Issues tab for scrcpy not sndcpy but i think the other one is pretty dead

I tried with different methods but nothing seams to work
could you guys help me modify the .bat so it remains hidden or something like that?

**Comment by McBluna** [comment]
see **scrcpy-noconsole.vbs** from scrcpy-win64-v2.0.zip as template

**Comment by YacuGitHub** [comment]
Could you make me a little guide please?

**Comment by TimmyP777** [comment]
Yeah any chance you could modify that vbs script so it waits for scrcpy and then kills adb. Adb is left running in background in all solutions I can find.

So it runs with batch script, with console, but it waits properly.

Running in vbs to hide console I cannot find a way to make it wait to close, then kill adb.

Entire feature seems kinda lukewarm if it leaves adb running.
